compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/KnownTargets.cpp

```diff
@@ -116,9 +117,12 @@ TargetAttr createTargetAttr(const TargetDetails &details, StringRef arch,
       DictionaryAttr{});
 
   TargetChipAttr targetChip;
-  if (details.chip)
-    targetChip =
-        TargetChipAttr::get(context, details.chip->wgpCount, DictionaryAttr{});
+  if (details.chip) {
+    StringAttr skuAttr =
+        StringAttr::get(context, details.chip->sku.value_or(""));
```

Comment:
```suggestion
    auto skuAttr =
        StringAttr::get(context, details.chip->sku.value_or(""));
```
The type is obvious based on the RHS: https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable